### PR TITLE
Adjusting text for intro, adding roadmap & planning files

### PR DESCRIPTION
**Pull request by willykaram**
I've made a fair number of revisions to the text/organization of some of the intro files. Part of the thinking here is that we need to cater to several audiences and not everyone is going to want all the technical details. For instance, many readers are just going to want to get to the examples/recipes and use this as a bit of a cookbook for how to implement media on their site. Whereas others are hopefully going to want to get more involved in contributing. I will add more revisions in the coming days/weeks and hopefully help to get an initial version of the guide done.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,51 @@
...
+##Introduction
+
+This guide provides and overview of the approach to Media in Drupal 8.
+
+The Drupal Media Team has set an ambitious [goal](roadmap.md) of ensuring Drupal offers best media solution available in any web framework. 
+
+We still have a ways to go to achieve that goal, but this guide will help you to understand our [Roadmap](roadmap.md) and [Module Status](modules/intro.md). 
+
+
+As discussed in the [Roadmap](roadmap.md) and throughout this Guide, the Media Team has worked collaboratively to separate Media functionality into several independent modules that can be used together based as appropriate. Overtime we'll be adding examples of how these modules can be integrated.
```
Comment (slashrsm) [inline README.md]: Should "Overtime" be "Over time"?

File: README.md [hunk 2]
```diff
@@ -1,10 +1,51 @@
...
+* [Entity Browser](modules/entity_browser/intro.md)
+* [File Entity](modules/file_entity/intro.md)
+* [Media Entity](modules/media_entity/intro.md)
+* [URL Embed](modules/url_embed/intro.md).
+
+
+
+### Troubleshooting
 
 If you have trouble understanding any part of the documentation, please notify those of us who work on this section by [creating an issue](https://github.com/drupal-media/handbook/issues) and clearly explain what you don't understand and why - we're happy to hear from you, your contribution helps everyone at Drupal!
```
Comment (slashrsm) [inline README.md:5]: This is deprecated link. Should we update to https://github.com/drupal-media/d8-guide/issues?

File: contributing.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+The Drupal Media Team has been growing over the last few years and gradually gaining momentum. We are always looking for new team members to help moving things forward and there are many ways to get involved, including including helping with funding, project management communications, documentation, testing and development
+
+##Funding
+
+*FPO/TBA - This content needs to be written, see our [Contributing](contributing.md) section get involved.*
+
+
+##Documentation
+
+You can also contribute directly on our [guide repository](https://github.com/drupal-media/handbook) by editing the files through the [GitHub](https://github.com/) interface directly in your browser. Alternatively, you can clone the repository and edit the guide in your favorite text editor or in the official [GitBook desktop app](https://www.gitbook.com/editor).
```
Comment (slashrsm) [inline contributing.md]: Old repo url again.

File: modules/intro.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+ * *Latest Status:* TBA
+
+
+* [**Flysystem:**](modules/flysystem/README.md) provides a bridge to use Flysystem as Drupal stream wrappers ([d.o](https://www.drupal.org/project/flysystem))
+ * *Latest Status:* TBA
+
+
+###D8 Media U/I Modules
+
+* [**Crop API:**](modules/crop/intro.md) an API for image cropping that's intended to be used with a related IP module such as [Image Widget Crop]( https://www.drupal.org/sandbox/woprrr/2571403) ([d.o](https://www.drupal.org/project/crop))
```
Comment (slashrsm) [inline modules/intro.md]: What does IP stand for?

File: modules/intro.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+ * *Latest Status:* TBA
+
+
+* [**URL Embed:**](https://www.drupal.org/project/url_embed) allows embedding of URLs using the Drupal 8 WYSIWYG or link fields ([d.o](https://www.drupal.org/project/url_embed))
+ * *Latest Status:* TBA
+
+
+###D8 Media Display Modules
+
+* [**Fallback Formatter:**](https://www.drupal.org/project/fallback_formatter) a field formatter that attempt multiple formatters for displaying file and applies the first one that returns output.
```
Comment (slashrsm) [inline modules/intro.md]: We should add Field formatter to this section (drupal.org/project/field_formatter).

File: roadmap.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
+#Drupal Media Roadmap
+
+This section of the Guide outlined the strategy behind Medi in Drupal 8, and our the goals and milestones on our roadmap.
```
Comment (slashrsm) [inline roadmap.md]: s/Medi/Media

File: roadmap.md [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+
+Apart from delivering great functionality, this strategy for D8 aims to allow for more collaboration, a faster release cycle, and greater flexbility in selecting which functionality/moduls are needed for a given project. Overall, this should make Media in Drupal easier to use, both for developers, site builders, as well as content editors and other end users.
+
+##Long Term Goals 
+
+The Drupal Media Team has set an ambitious goal of making Drupal the best media handling solution available in any web framework.
+
+If we succeed in accomplishing this goal, ideally the approach and lessons learned can be useful for other open source web frameworks too (such as [Wordpress](http://wordpress.org), [Django](https://www.djangoproject.com/), [Phoenix](http://www.phoenixframework.org), and others).
+
+We still have a long way to go to achieve that goal.  However, the plans below will help you to understand the roadmap that we're following to achieve that goal, and the [Status](Status.md) section of this Guide will keep you up-to-date on our progress along that roadmap.
```
Comment (slashrsm) [inline roadmap.md]: Extra space after first sentence.

**Comment by slashrsm** [comment]
Thank you! This looks awesome. I left few minor comments.

**Comment by willykaram** [comment]
Thanks for the review/comments. I just corrected each of the issues.

**Comment by willykaram** [comment]
Do you want me to squash the commits, or keep them separate for review purposes?

**Comment by slashrsm** [comment]
Yup, squash and rebase against master would be great. There is one (minor) thing left.

**Pull request closed without merging**